source/index.ts

```diff
@@ -68,7 +68,7 @@ const defaults: InstanceDefaults = {
 		// TODO: Set this to `true` when Got 12 gets released
 		http2: false,
 		allowGetBody: false,
-		rejectUnauthorized: true,
+		https: undefined,
```

Comment:
Nope. Please define this as object with `rejectUnauthorized` set to `true` :)

Reply:
The problem is that `https.rejectUnauthorized` is going to override `rejectUnauthorized` if a used is using `rejectUnauthorized`

Reply:
I'm fine with that. Show 'em a deprecation warning then :)